src/test/java/uk/gov/companieshouse/presenter/account/controller/PresenterAccountControllerTest.java

```diff
@@ -98,6 +107,26 @@ void testThrowExcetionInternalError() {
         assertThat(response.getStatusCode(), is(HttpStatus.INTERNAL_SERVER_ERROR));
     }
 
+    @Test
```

Comment:
In this class can we add a verify to the 201 created test to check the send presenter id function was called. Just so we have a positive journey test for the kafka stuff